### PR TITLE
FLAG-73: upgrade openmrs version from 1.9.9 to 2.0.0

### PR DESCRIPTION
**Pull request by ManojLL**
### Description of what I change
Upgrade the openMRS platform version from 1.9.9 to 2.0.0

Issue ### Worked On
worked on [FLAG-73](https://openmrs.atlassian.net/browse/FLAG-73)

[FLAG-73]: https://openmrs.atlassian.net/browse/FLAG-73?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by wikumChamith — CHANGES_REQUESTED** [review]

File: omod/pom.xml [hunk 1]
```diff
@@ -72,6 +72,22 @@
...
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>legacyui-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>legacyui-omod</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
```
[the remark this answers is not in the document]
Comment (ManojLL) [reply omod/pom.xml:85]: yes, without adding these dependencies we can not use legacy ui things in controllers.

File: pom.xml [hunk 2]
```diff
@@ -107,18 +107,37 @@
...
-		<openMRSMinorVersion>1.9</openMRSMinorVersion>
+		<openMRSVersion>2.0.0</openMRSVersion>
+		<openMRSMinorVersion>2.0</openMRSMinorVersion>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<uiframeworkVersion>3.11.0</uiframeworkVersion>
 		<metadatadeployVersion>1.8.1</metadatadeployVersion>
 		<metadatasharingVersion>1.2.2</metadatasharingVersion>
 		<webservices.rest-omod.version>2.40.0</webservices.rest-omod.version>
+		<legacyuiVersion>1.2.3</legacyuiVersion>
+		<omodHbmConfig>
```
[the remark this answers is not in the document]
Comment (ManojLL) [reply pom.xml:135]: we had a profile for platform 2.0.0, in there contains these properties for platform 2.0.0  so just add them to here.

[the remark this answers is not in the document]
Comment (wikumChamith) [reply pom.xml:135]: Do we get any errors without "javax.servlet-api" ?

File: omod/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (wikumChamith) [reply omod/pom.xml:85]: Do we get any errors without "javax.servlet-api" ?

File: pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (wikumChamith) [reply pom.xml:135]: Do we get any errors if we don't include these?

[the remark this answers is not in the document]
Comment (ManojLL) [reply pom.xml]: module build successfully without this properties.

File: omod/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ManojLL) [reply omod/pom.xml]: we can remove "javax.servlet-api" and I updated the changes

File: pom.xml [hunk 3]
```diff
@@ -107,18 +107,31 @@
 				<version>${metadatasharingVersion}</version>
 				<scope>test</scope>
 			</dependency>
+			<dependency>
```
[the remark this answers is not in the document]
Comment (ManojLL) [reply pom.xml:110]: I removed legacyui-api dependency

File: pom.xml [hunk 4]
```diff
@@ -107,18 +107,31 @@
...
-		<openMRSVersion>1.9.9</openMRSVersion>
-		<openMRSMinorVersion>1.9</openMRSMinorVersion>
+		<openMRSVersion>2.0.0</openMRSVersion>
+		<openMRSMinorVersion>2.0</openMRSMinorVersion>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<uiframeworkVersion>3.11.0</uiframeworkVersion>
 		<metadatadeployVersion>1.8.1</metadatadeployVersion>
 		<metadatasharingVersion>1.2.2</metadatasharingVersion>
 		<webservices.rest-omod.version>2.40.0</webservices.rest-omod.version>
+		<legacyuiVersion>1.2.3</legacyuiVersion>
```
Comment (wikumChamith) [inline pom.xml:134]: Did you try using a more recent version of legacyui?

Comment (ManojLL) [reply pom.xml]: I updated it to latest legacy UI version

**Pull request merged**